api/staleness.py

```diff
@@ -49,26 +56,54 @@ def _validate_input_data(body):
         abort(HTTPStatus.BAD_REQUEST, f"Validation Error: {str(e.messages)}")
 
 
-def _update_hosts_staleness_async(org_id, app):
+def _update_hosts_staleness_async(identity, app, staleness):
     with app.app_context():
+        threadctx.request_id = None
         logger.debug("Starting host staleness update thread")
         try:
-            logger.info(f"Querying hosts for org_id: {org_id}")
-            hosts_query = Host.query.filter(Host.org_id == org_id)
+            logger.info(f"Querying hosts for org_id: {identity.org_id}")
+            hosts_query = Host.query.filter(Host.org_id == identity.org_id)
             num_hosts = hosts_query.count()
+            st = staleness_timestamps()
+            staleness = serialize_staleness_to_dict(staleness)
```

Comment:
Better to make a new var instead of overwriting the param imo, maybe something like this
```suggestion
            staleness_dict = serialize_staleness_to_dict(staleness)
```

Reply:
Done!